Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM docker.io/library/rockylinux:9@sha256:d7be1c094cc5845ee815d4632fe377514ee6ebcf8efaed6892889657e5ddaaa6 as rockylinux9
 FROM docker.io/library/golang:1.20.0-bullseye as golang_120
 FROM docker.io/library/golang:1.21.0-bullseye as golang_121
-FROM docker.io/library/node:22.3.0-bullseye as node_223
+FROM docker.io/library/node:22.5.1-bullseye as node_223
```

Comment:
Do we depend on specific version of node?

If yes, then we cannot merge it
If no, then it should be renamed `as node`